helm/values-uat.yaml

```diff
@@ -52,6 +52,8 @@ microservice-chart:
       PAYMENT_TOKEN_VALIDITY_TIME: "1800"
       TRANSACTIONS_RETRY_OFFSET: "60"
       CLOSURE_RETRY_INTERVAL: "30"
+      WEB_LOGGING_LEVEL: "off"
+      PAGOPA_LOGGING_LEVEL: "info"
```

Comment:
What about `PAGOPA_LOGGING_LEVEL`?

Reply:
I put here pagopa referring the application package name 'it.pagopa...'. Maybe with this consideration in mind we can update it to `TRANSACTION_LOGGING_LEVEL` maybe

Reply:
Yeah even `TRANSACTIONS_LOGGING_LEVEL` would be ok. From an outsider point of view it should be understandable what each environment variable does, `PAGOPA_LOGGING_LEVEL` ties a little too much into an implementation detail (maybe even `SERVICE_LOGGING_LEVEL` or `BUSINESS_LOGGING_LEVEL` would be ok I guess)

Reply:
I like `SERVICE_LOGGING_LEVEL`.
Refactored to SERVICE_LOGGING_LEVEL into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/119/commits/e31c870d5011dc8fe441642515c0b6d0c93daca9